package/network/services/hostapd/files/hostapd.sh

```diff
@@ -490,7 +490,7 @@ hostapd_set_bss_options() {
 	set_default uapsd 1
 	set_default wpa_disable_eapol_key_retries 0
 	set_default tdls_prohibit 0
-	set_default eapol_version 0
+	set_default eapol_version $(($wpa & 1))
```

Comment:
IIRC, this triggers a shellcheck warning. There's no need for $ in $(())

Reply:
Agreed, but this was-deliberate/intentional as this is the pattern used elsewhere in the file. Should we change the others?

Reply:
See from wpa_supplicant_add_rate():

local val="$(($2 / 1000))"
local sub="$((($2 / 100) % 10))"

And from hostapd_append_wep_key():

zidx=$(($idx - 1))

Do you want all these changing in this commit for consistency?

Reply:
Just change the wpa line. No need for the others for now.

Reply:
@nicklowe Just imagine how it started with 1 "wrong" one and all the others then adding "wrong" ones as well for consistency. Consistent style is a good thing when it's about matters of taste (wrapping lines, indent, etc.), but in this case there is a clear preference for a particular version.

However, as neheb told, no need to touch the old stuff, at least not in the same commit.